app/components/ChartSettings/modules/Metadata.js

```diff
@@ -39,10 +51,13 @@ export default class Metadata extends Component {
   };
 
   render() {
+    if (!this.shouldShowMetadata.subtitle) {
+      delete this.state.subtitle;
```

Comment:
Went with this suggestion as well, but the ternary was giving ESLint headaches. Couldn't decide what it wanted - kept yelling at me if I added a block statement, took away the block statement, used return, didn't use return. I went with an `if/else` block to keep it quiet.